modules/grandpa/src/benchmarking.rs

```diff
@@ -47,15 +47,14 @@
 use crate::*;
 
 use bp_test_utils::{
-	accounts, authority_list, make_justification_for_header, test_keyring, JustificationGeneratorParams, ALICE,
```

Comment:
I can't comment on lines that weren't changed, but lines 42-45 are outdated now. Can you update it with the final calculation strategy you've outlined (base weight + precommit + ancestry) in the Grandpa pallet?

Reply:
Updated